Moco/View/StoryView.swift

```diff
@@ -7,10 +7,36 @@
 
 import SwiftUI
 
+struct Narrative {
+    var text: String
+
+    var duration: Double // MARK: In seconds
+
+    var positionX: Double // MARK: position in percentage of the size of the screen
+
+    var positionY: Double
+}
+
 struct StoryView: View {
+    // MARK: - Environments stored property
+
+    @Environment(\.timerViewModel) private var timerViewModel
+    @Environment(\.audioViewModel) private var audioViewModel
     @Environment(\.navigate) private var navigate
+    @EnvironmentObject var speechViewModel: SpeechRecognizerViewModel
```

Comment:
blm diinclude in di env

Reply:
ini soalnya kalo dari simulator dia error makanya di disable di simulator